react/features/base/participants/functions.js

```diff
@@ -244,21 +260,23 @@ export function getParticipantCountWithFake(stateful: Object | Function) {
 /**
  * Returns participant's display name.
  *
- * @param {(Function|Object)} stateful - The (whole) redux state, or redux's
- * {@code getState} function to be used to retrieve the state.
+ * @param {(Function|Object)} stateful - The (whole) redux state, or redux's {@code getState} function to be used to
+ * retrieve the state.
  * @param {string} id - The ID of the participant's display name to retrieve.
  * @returns {string}
  */
-export function getParticipantDisplayName(
-        stateful: Object | Function,
-        id: string) {
+export function getParticipantDisplayName(stateful: Object | Function, id: string) {
     const participant = getParticipantById(stateful, id);
     const {
         defaultLocalDisplayName,
         defaultRemoteDisplayName
     } = toState(stateful)['features/base/config'];
 
     if (participant) {
+        if (participant.isFakeScreenShareParticipant) {
```

Comment:
I think we can move towards a more functional approach to determining if a participant is a `virtualScreenShareParticipant` with just an `id`. I will look into only storing ids instead of entire participants in a follow up refactor PR